### PR TITLE
Improve giveaways

### PR DESCRIPTION
**Pull request by mzampetakis**
Just limited lines up to 80 characters. Content is up to date.

### CODE REVIEW

File: Giveaways.md [hunk 1]
```diff
@@ -1,15 +1,36 @@
 # Giveaways
-From time to time DevStaff has the opportunity to offer some giveaways during the meetups. The prizes are provided by [sponsors](https://github.com/devstaff-crete/DevStaff-Heraklion/blob/master/Sponsors.md) in return of advertising.
+From time to time DevStaff has the opportunity to offer some giveaways during 
+the meetups. The prizes are provided by [sponsors]
```
Comment (DaKnOb) [inline Giveaways.md:3]: Meetups

File: Giveaways.md [hunk 2]
```diff
@@ -1,15 +1,36 @@
...
+(https://github.com/devstaff-crete/DevStaff-Heraklion/blob/master/Sponsors.md) 
+in return of advertising.
```
Comment (DaKnOb) [inline Giveaways.md:5]: in exchange for promotion.

File: Giveaways.md [hunk 3]
```diff
@@ -1,15 +1,36 @@
...
 
 The process, the tools and the rules we use for these giveaways are described here.
 
-# Prizes list
-All prizes will be written down at each meetup's announcement at [meetup.com](https://www.meetup.com/devstaff/). Moreover, at the intro presentation, when sponsors are presented, each prize will be explicitly mentioned along with the type, the amount of prizes and the sponsor of each one.
+## Prizes list
+All prizes will be written down at each meetup's announcement at [meetup.com]
```
Comment (DaKnOb) [inline Giveaways.md:10]: Meetup

File: Giveaways.md [hunk 4]
```diff
@@ -1,15 +1,36 @@
...
+(https://www.meetup.com/devstaff/). Moreover, at the intro presentation, when 
+sponsors are presented, each prize will be explicitly mentioned along with the 
+type, the amount of prizes and the sponsor of each one.
```
Comment (DaKnOb) [inline Giveaways.md:13]: `/s/and/and,/`

I think you can also remove the "of each one".

**Review by DaKnOb — CHANGES_REQUESTED** [review]

File: Giveaways.md (hunk 1 above)
Comment (mzampetakis) [reply Giveaways.md]: Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/494/commits/d5c520d78b77eee7c7e05dc4df9ee9a3792c41f3

File: Giveaways.md [hunk 5]
```diff
@@ -1,15 +1,36 @@
...
 
-# Participants eligibility
-All participants of a meetup are eligible for the raffling process including the speaker(s), attendees, and org team members. The participants of a meetup are the members that have joined our meetup's place (in case of physical meetups) or our communication tool (in case of remote meetups).
+## Participants eligibility
+All participants of a meetup are eligible for the raffling process including
```
[the remark this answers is not in the document]
Comment (mzampetakis) [reply Giveaways.md]: Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/494/commits/d5c520d78b77eee7c7e05dc4df9ee9a3792c41f3

File: Giveaways.md [hunk 6]
```diff
@@ -1,15 +1,36 @@
...
 
-# Raffle Process
-During the meetup (especially during the intro and at the closing phase) a link (to Google Forms) will be shared where everyone present will be able to add their name & email in order to participate to the raffling process. A number will be assigned to each participant's entry and then we shall use the [random.org](https://www.random.org/) to generate one number per giveaway prize. The list with the participants alongside the assigned unique number will be presented before the giveaway begins. Each participant's unique number corresponding to the random number produced will win the prize. All prizes will be given in the order mentioned at the top of the page in the provided link where participants will fill their details. All details (name and emails) as well as the prizes' winners will be stored for 1 year after the meetup's date.
+## Raffle Process
+During the meetup (especially during the intro and at the closing phase) a 
+link (to Google Forms) will be shared where everyone present will be able to 
+add their name & email in order to participate to the raffling process. A 
+number will be assigned to each participant's entry and then we shall use the
```
[the remark this answers is not in the document]
Comment (mzampetakis) [reply Giveaways.md]: Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/494/commits/d5c520d78b77eee7c7e05dc4df9ee9a3792c41f3

File: Giveaways.md [hunk 7]
```diff
@@ -1,15 +1,36 @@
...
+will be presented before the giveaway begins. Each participant's unique number 
+corresponding to the random number produced will win the prize. All prizes 
+will be given in the order mentioned at the top of the page in the provided 
+link where participants will fill their details. All details (name and emails) 
+as well as the prizes' winners will be stored for 1 year after the meetup's 
+date.
 
-The whole raffle process described at this step will take place during the end of the meetup (live or) in front of all participants.
+The whole raffle process described at this step will take place during the end 
+of the meetup (live or) in front of all participants.
```
[the remark this answers is not in the document]
Comment (mzampetakis) [reply Giveaways.md]: Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/494/commits/d5c520d78b77eee7c7e05dc4df9ee9a3792c41f3

File: Giveaways.md [hunk 8]
```diff
@@ -1,15 +1,36 @@
...
+link (to Google Forms) will be shared where everyone present will be able to 
+add their name & email in order to participate to the raffling process. A 
+number will be assigned to each participant's entry and then we shall use the 
+[random.org](https://www.random.org/) to generate one number per giveaway 
+prize. The list with the participants alongside the assigned unique number 
+will be presented before the giveaway begins. Each participant's unique number 
+corresponding to the random number produced will win the prize. All prizes 
+will be given in the order mentioned at the top of the page in the provided 
+link where participants will fill their details. All details (name and emails) 
+as well as the prizes' winners will be stored for 1 year after the meetup's
```
[the remark this answers is not in the document]
Comment (mzampetakis) [reply Giveaways.md]: Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/494/commits/d5c520d78b77eee7c7e05dc4df9ee9a3792c41f3

File: Giveaways.md (hunk 2 above)
Comment (mzampetakis) [reply Giveaways.md]: Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/494/commits/d5c520d78b77eee7c7e05dc4df9ee9a3792c41f3

File: Giveaways.md (hunk 3 above)
Comment (mzampetakis) [reply Giveaways.md]: Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/494/commits/d5c520d78b77eee7c7e05dc4df9ee9a3792c41f3

File: Giveaways.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (mzampetakis) [reply Giveaways.md]: Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/494/commits/e2b786bf58aee532184c5cb1cf8699163e3d392e

File: Giveaways.md [hunk 9]
```diff
@@ -1,15 +1,36 @@
...
+add their name & email in order to participate to the raffling process. A 
+number will be assigned to each participant's entry and then we shall use the 
+[random.org](https://www.random.org/) to generate one number per giveaway 
+prize. The list with the participants alongside the assigned unique number 
+will be presented before the giveaway begins. Each participant's unique number 
+corresponding to the random number produced will win the prize. All prizes 
+will be given in the order mentioned at the top of the page in the provided 
+link where participants will fill their details. All details (name and emails) 
+as well as the prizes' winners will be stored for 1 year after the meetup's 
+date.
```
[the remark this answers is not in the document]
Comment (mzampetakis) [reply Giveaways.md]: Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/494/commits/1b3310329ad88ade5b0e67c40a2432bcd520dece

**Review by DaKnOb — APPROVED** [review]

**Review by kouts — APPROVED** [review]

**Pull request merged**